src/app/charts/core/subsys-deps.yaml

```diff
@@ -171,6 +171,10 @@ settings:
       url: ARKCASE_REPORTS_ADMIN_URL
       username: ARKCASE_REPORTS_ADMIN_USERNAME
       password: ARKCASE_REPORTS_ADMIN_PASSWORD
+      group:
+        env:
+          name: ARKCASE_REPORTS_ADMIN_GROUP
+          optional: true
```

Comment:
You can replace this with just `group: ARKCASE_REPORTS_ADMIN_GROUP` because the other values in the secret are already mandatory, and the secret will never not contain that value.

Thus, you're as safe assuming it'll always be there as are the other values.